shared/login/signup/success/index.render.desktop.js

```diff
@@ -22,7 +22,7 @@ export default class Render extends Component {
 
   render () {
     return (
-      <Container onBack={this.props.onBack} style={styles.container}>
```

Comment:
we could just keep container (and the styling) so its more consistent and if onBack is null it just doens't render the back button